examples/IPC/ui/AddressBookIPCServerLocalQMLModel.qml

```diff
@@ -57,7 +57,7 @@ Item {
         id: proxy
     }
 
-    AddressBookImplementationBase {
+    AddressBook {
```

Comment:
I don't think this is correct, because the implementation is provided here indeed and the implementation should be based on AddressBookImplementationBase.

Reply:
yes you're right , its wrong  i mistyped it somehow .